projects/components/src/breadcrumbs/breadcrumbs.component.test.ts

```diff
@@ -53,12 +57,21 @@ describe('BreadcrumbsComponent', () => {
     expect(spectator.queryAll('.breadcrumbs .breadcrumb').length).toBe(3);
   });
 
-  it('should show icon only on the first crumb', () => {
+  it('should show icon on the first crumb', () => {
     const crumbs = spectator.queryAll('.breadcrumbs .breadcrumb');
     expect(crumbs[0].querySelector('ht-icon')).toExist();
+  });
 
+  it('should display the icon if alwaysShowIcon is true', () => {
+    const crumbs = spectator.queryAll('.breadcrumbs .breadcrumb');
     crumbs.shift();
-    crumbs.forEach(crumb => expect(crumb.querySelector('ht-icon')).toBeNull());
+    crumbs.forEach((crumb, index) => {
```

Comment:
I find this test a little confusing to read because of the preceding `shift` - could we remove that so the index being compared matches the input test data.

Reply:
Replaced it